### PR TITLE
Add more validations in matrimony profile form

### PR DESCRIPTION
**Pull request by bhawanabadlani**

### CODE REVIEW

File: vmb/matrimony/forms.py [hunk 1]
```diff
@@ -499,6 +502,19 @@ def clean(self):
...
         ):
             raise forms.ValidationError(_("Select a valid place from maps"))
+        brothers_married = self.cleaned_data["brothers_married"]
+        brothers = self.cleaned_data["brothers"]
+        sisters_married = self.cleaned_data["sisters_married"]
+        sisters = self.cleaned_data["sisters"]
+
+        if brothers_married > brothers:
+            raise forms.ValidationError(
+                _("Brothers married cannot be greater than no. of brothers")
```
Comment (rtnpro) [inline vmb/matrimony/forms.py]: use full word `number` instead of `no.`

**Review by rtnpro — CHANGES_REQUESTED** [review]

**Comment by rtnpro** [comment]
@bhawanabadlani You commit messages don't make sense: 
image

Each commit message to explicitly mention the changes made, not some vague phrases.

File: vmb/matrimony/forms.py (hunk 1 above)
Comment (bhawanabadlani) [reply vmb/matrimony/forms.py]: ok

File: vmb/matrimony/forms.py [hunk 2]
```diff
@@ -590,12 +606,23 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = Layout(
             Submit("submit", "Next" if self.wizard else "Save"),
         )
+        self.fields.required = True
```
Comment (rtnpro) [inline vmb/matrimony/forms.py:633]: Why do we need this? @bhawanabadlani

**Review by rtnpro — APPROVED** [review]

**Pull request merged**